parlai/tasks/cmu_dog/build.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+import parlai.core.build_data as build_data
+import os
+from parlai.core.build_data import DownloadableFile
+
+RESOURCES = [
+    DownloadableFile(
+        'http://parl.ai/downloads/cmu_dog/cmu_dog.tar.gz',
+        'cmu_dog.tar.gz',
+        '30d2bac0dae6b4e4c0b94ba581fffaf1acb46838480f7ad6736ad03d9312ae9d',
```

Comment:
messaged offline about this but we should be downloading the dataset from the source, and building the splits ourselves (in the `build` function below)